Filtered_kernel/include/CGAL/internal/Static_filters/Do_intersect_3.h

```diff
@@ -265,6 +264,7 @@ class Do_intersect_3
         if (double_tmp_result < -eps)
           return true;
         else {
+          // return true; // AF:  just to test conservative approach
```

Comment:
@lrineau   Maybe you have an idea for the following problem:  We use this  sphere/bbox   test in the AABB tree for finding the closest triangle.  While vtuning code I noticed that there are filter failures that is we do the test with exact arithmetic.   I could well live with a `true` (that is overlap) in case of uncertainly.  The question is, if we have to add such a functor to the kernel, or if we can add somehow a parameter.

Reply:
That would be a sort of "possibly intersect" instead of "do intersect".

`CGAL::internal::Static_filters_predicates::Do_intersect_3` could have a third template parameter (a non-type template argument, I suggest a dedicated enum, to avoid [the curve of a Boolean parameter](https://softwareengineering.stackexchange.com/a/147983/118556)), with a default-value.

Then we can add a new functor name in the Kernel concept, and it would be implemented using either `Do_intersect_3` or `CGAL::internal::Static_filters_predicates::Do_intersect_3` with the special value as third template argument.


I might actually be more efficient to implement that `possibly_intersect(sphere, bbox)` by `do_overlap(sphere.bbox(), bbox)`.

Reply:
Note that I also mentioned the tag this morning with https://github.com/CGAL/cgal/pull/4654 in mind where we also brought the tag onto the table to allow/forbid degenerate input.

Reply:
To use the bbox of the sphere makes it 25% slower.